privacyidea/api/token.py

```diff
@@ -800,9 +799,9 @@ def tokenrealm_api(serial=None):
         realm_list = [r.strip() for r in realms.split(",")]
     g.audit_object.log({"serial": serial})
 
-    res = set_realms(serial, realms=realm_list)
+    set_realms(serial, realms=realm_list)
     g.audit_object.log({"success": True})
-    return send_result(res == 1)
+    return send_result(True)
```

Comment:
This does also not seem quite consistent to me.
In this case we always add ``"success": True`` to the audit. Setting an empty list would remove the realms?

Reply:
Hmm, I'm not sure here: ``set_realms`` only gets a serial, not a user, so it will raise a ``ResourceNotFound`` if the token with the serial does not exist. So if the line after ``set_realms`` is executed, I would say the request is successful?